project_wbs/view/account_analytic_account_view.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_account_analytic_account_list" model="ir.ui.view">
+        <field name="name">account.analytic.account.list</field>
+        <field name="model">account.analytic.account</field>
+        <field name="type">tree</field>
+        <field eval="8" name="priority"/>
+        <field name="inherit_id" ref="analytic.view_account_analytic_account_list"/>
+        <field name="arch" type="xml">
+            <field name="code" position="attributes">
+                <attribute name="invisible">True</attribute>
+            </field>
+
+            <field name="partner_id" position="after">
+                <field name="code"/>
+                <field name="user_id" invisible="1"/>
+                <field name="manager_id"/>
```

Comment:
Manager_id is only available in tree view, thus there is no way to register the manager.

Reply:
You are right, this is already fixed. Thanks for reviewing.